ta4j-core/src/main/java/org/ta4j/core/num/Num.java

```diff
@@ -41,271 +41,286 @@
-
-    /**
-     * Checks if this value is less than another.
-     * 
-     * @param other the other value, not null
-     * @return true is this is less than or equal the specified value, false
-     *         otherwise
-     */
-    boolean isLessThanOrEqual(Num other);
-
-    /**
-     * Returns the minimum of this {@code num} and {@code other}.
-     * 
-     * @param other value with which the minimum is to be computed
-     * @return the {@code num} whose value is the lesser of this {@code num} and
-     *         {@code other}. If they are equal, method, {@code this} is returned.
-     */
-    Num min(Num other);
-
-    /**
-     * Returns the maximum of this {@code num} and {@code other}.
-     * 
-     * @param other value with which the maximum is to be computed
-     * @return the {@code num} whose value is the greater of this {@code num} and
-     *         {@code other}. If they are equal, method, {@code this} is returned.
-     */
-    Num max(Num other);
-
-    /**
-     * Returns the {@link Function} to convert a number instance into the
-     * corresponding Num instance
-     * 
-     * @return function which converts a number instance into the corresponding Num
-     *         instance
-     */
-    Function<Number, Num> function();
-
-    /**
-     * Transforms a {@link Number} into a new Num instance of this <code>Num</code>
-     * implementation
-     * 
-     * @param value the Number to transform
-     * @return the corresponding Num implementation of the <code>value</code>
-     */
-    default Num numOf(Number value) {
-        return function().apply(value);
-    }
-
-    /**
-     * Transforms a {@link String} into a new Num instance of this with a precision
-     * <code>Num</code> implementation
-     * 
-     * @param value     the String to transform
-     * @param precision the precision
-     * @return the corresponding Num implementation of the <code>value</code>
-     */
-    default Num numOf(String value, int precision) {
-        MathContext mathContext = new MathContext(precision, RoundingMode.HALF_UP);
-        return this.numOf(new BigDecimal(value, mathContext));
-    }
-
-    /**
-     * Only for NaN this should be true
-     * 
-     * @return false if this implementation is not NaN
-     */
-    default boolean isNaN() {
-        return false;
-    }
-
-    /**
-     * Converts this {@code num} to a {@code double}.
-     * 
-     * @return this {@code num} converted to a {@code double}
-     */
-    default double doubleValue() {
-        return getDelegate().doubleValue();
-    }
-
-    default int intValue() {
-        return getDelegate().intValue();
-    }
-
-    default long longValue() {
-        return getDelegate().longValue();
-    }
-
-    default float floatValue() {
-        return getDelegate().floatValue();
-    }
-
-    @Override
-    int hashCode();
-
-    @Override
-    String toString();
-
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    boolean equals(Object obj);
+	/**
```

Comment:
Please revert the file indentation change. You used tab for indentation, but the whole project uses 4 spaces (which is recommended).

Reply:
Issue not resolved